documentation/docs/strategies/advanced-mm/order-override.md

```diff
@@ -0,0 +1,42 @@
+# Order Override
+
+This feature allows users to customize and specify how Hummingbot creates orders in terms of order levels, spread, and size.
+
+
+## How It Works
+
+Since this is a feature designed for advanced users, it is not configurable from the Hummingbot client. Follow the steps below to set up `order_override` parameter:
+
+1. Edit the strategy config file located in the Hummingbot `conf` folder using a text editor.</br>See [Where are my config and log files?](faq/troubleshooting/#where-are-my-config-and-log-files) in the FAQ page for more information.
+1. The input should be in a dictionary format and the key is user-defined.</br>Make sure there is a space between the colon ( : ) and open bracket ( [ ) as shown in the [Sample Configuration](#sample-configuration) then save your changes.
+1. If Hummingbot is running, run `exit` command and restart so the changes will take effect
```

Comment:
You can also `stop` then `import` the config file again and `start`.

Reply:
Thanks @Nullably, done making the changes.

![image](https://user-images.githubusercontent.com/50150287/95036510-c4e71000-06fa-11eb-885f-ce02388dad96.png)